<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOrEditTagDialog</class>
 <widget class="QDialog" name="AddOrEditTagDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="tagNameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>tagNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="tagNameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="parentTagName">
     <property name="text">
      <string>Parent tag:</string>
     </property>
     <property name="buddy">
      <cstring>parentTagNameComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="statusBar"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="parentTagNameComboBox"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tagNameLineEdit</tabstop>
  <tabstop>parentTagNameComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddOrEditTagDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddOrEditTagDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
